stages/test/conftest.py

```diff
@@ -0,0 +1,43 @@
+import pathlib
+from types import ModuleType
+
+import pytest
+
+import osbuild.meta
+from osbuild.testutil.imports import import_module_from_path
+
+
+@pytest.fixture
+def stage_module(request: pytest.FixtureRequest) -> ModuleType:
+    """stage_module is a fixture that imports a stage module by its name
+
+    This fixture must be indirectly parametrized with the stage name string.
+    The stage name string is used to find the stage module in the caller's
+    parent directory.
+    """
+    if not hasattr(request, "param") or not isinstance(request.param, str):
+        raise ValueError("stage_module fixture must be indirectly parametrized with the stage name string")
+
+    caller_dir = pathlib.Path(request.node.fspath).parent
+    module_path = caller_dir.parent / request.param
+    return import_module_from_path("stage", str(module_path))
```

Comment:
(super nitpick) `os.fspath(module_path)` would be the "canonical" way to transform a pathlib.Path to a "string" (at least AIUI). But this is will work on just fine .

Reply:
done